sdk/tables/azure-data-tables/tests/test_table_client_async.py

```diff
@@ -0,0 +1,540 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import unittest
+import pytest
+import platform
+
+from azure.data.tables.aio import TableServiceClient, TableClient
+from azure.data.tables._version import VERSION
+from devtools_testutils import ResourceGroupPreparer, StorageAccountPreparer
+
+from _shared.testcase import GlobalStorageAccountPreparer, TableTestCase
+
+from azure.core.exceptions import HttpResponseError
+# ------------------------------------------------------------------------------
+SERVICES = {
+    #TableServiceClient: 'table',
+    #TableClient: 'table',
+    #TableServiceClient: 'cosmos',
+    #TableClient: 'cosmos',
+}
+
+_CONNECTION_ENDPOINTS = {'table': 'TableEndpoint', 'cosmos': 'TableEndpoint'}
+
+_CONNECTION_ENDPOINTS_SECONDARY = {'table': 'TableSecondaryEndpoint', 'cosmos': 'TableSecondaryEndpoint'}
+
+class StorageTableClientTest(TableTestCase):
+    def setUp(self):
+        super(StorageTableClientTest, self).setUp()
+        self.sas_token = self.generate_sas_token()
+        self.token_credential = self.generate_oauth_token()
+
+    # --Helpers-----------------------------------------------------------------
+    def validate_standard_account_endpoints(self, service, account_name, account_key):
+        self.assertIsNotNone(service)
+        self.assertEqual(service.account_name, account_name)
+        self.assertEqual(service.credential.account_name, account_name)
+        self.assertEqual(service.credential.account_key, account_key)
+        self.assertTrue(
+            ('{}.{}'.format(account_name, 'table.core.windows.net') in service.url) or
+            ('{}.{}'.format(account_name, 'table.cosmos.azure.com') in service.url))
+        self.assertTrue(
+            ('{}-secondary.{}'.format(account_name, 'table.core.windows.net') in service.secondary_endpoint) or
+            ('{}-secondary.{}'.format(account_name, 'table.cosmos.azure.com') in service.secondary_endpoint))
+
+    # --Direct Parameters Test Cases --------------------------------------------
+    @GlobalStorageAccountPreparer()
+    async def test_create_service_with_key_async(self, resource_group, location, storage_account, storage_account_key):
+        # Arrange
+
+        for client, url in SERVICES.items():
```

Comment:
can you check that these tests are actually running as expected? I see SERVICES commented out above so I don't think we're actually hitting any of these calls/asserts.

Reply:
You're correct, we are not hitting any of these.

Reply:
is that intentional?

Reply:
discussed offline